templates/README_guide.md

```diff
@@ -53,6 +53,8 @@ When relevant, please provide [BCP-47 codes](https://tools.ietf.org/html/bcp47),
 
 ## Dataset Structure
 
+IMPORTANT: This section can to some part be generated automatically as described in the end of the file [here](#create-dataset-structure).
```

Comment:
If I understand correctly the tool is quite powerful when you have many configs and splits right ?
Then maybe for the general usage it's probably simpler to just write the paragraph manually.
Could we make it more explicit that this is for for a a more advanced usage and that it's optional ?
Let me know what you think

Reply:
Definitely! Feel free to go into the PR and rewrite it to whatever you think better fits the library :-)